04_model_inference.ipynb

```diff
@@ -116,23 +117,79 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 4,
+   "execution_count": 137,
+   "id": "54f19596-766e-4b5e-ac23-f8da626af6f2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def time_output(input_time):\n",
+    "    \n",
+    "    day1 = ''\n",
+    "    day2 = ''\n",
+    "    splitted = input_time.split(' - ')\n",
+    "    start = splitted[0]\n",
+    "    end = splitted[1]\n",
+    "    \n",
+    "    item1 = start.split(\", \")\n",
+    "    item2 = end.split(\", \")\n",
+    "    \n",
+    "    if len(item1) == 2:\n",
+    "        day1 = start.split(\", \")[0]\n",
+    "        time1 = start.split(\", \")[1]\n",
+    "    else:\n",
+    "        time1 = start.split(\", \")[0]\n",
+    "    \n",
+    "    h1,m1,s1 = re.split(':', time1)\n",
+    "\n",
+    "    if len(item2) == 2:\n",
+    "        day2 = end.split(\", \")[0]\n",
+    "        time2 = end.split(\", \")[1]\n",
+    "    else:\n",
+    "        time2 = end.split(\", \")[0]\n",
+    "    \n",
+    "    h2,m2,s2 = re.split(':', time2)\n",
+    "\n",
+    "    if h1:\n",
+    "        h1 = h1+\" hrs\"\n",
+    "    else:\n",
+    "        h1 = ''\n",
+    "    if h2:\n",
+    "        h2 = h2+\" hrs\"\n",
+    "    else:\n",
+    "        h2 = ''\n",
+    "    if m1:\n",
+    "        m1 = m1+\" mins\"\n",
+    "    else:\n",
+    "        m1 = ''\n",
+    "    if m2:\n",
+    "        m2 = m2+\" mins\"\n",
+    "    else:\n",
+    "        m2 = ''\n",
+    "\n",
+    "    s = \"From the time of opening of this PR, this PR is predicted to merge in the range {} {} {} and {} {} {} by our model\".format(day1,h1,m1,day2,h2,m2)\n",
```

Comment:
```suggestion
    "    s = \"Our model predicts that it will take between  {} {} {} and {} {} {} for this PR to be merged from the time it was opened .\".format(day1,h1,m1,day2,h2,m2)\n",
```

Reply:
thanks @MichaelClifford! made the change